### DISCUSSION TITLE
Welcome to cg-use-everywhere Discussions! Read this first...

### DISCUSSION THREAD

**Discussion by chrisgoringe** [chrisgoringe/cg-use-everywhere#158] [Announcements]
# Welcome

Welcome to the UE discussions! Please choose the right category...

## Ideas

Ideas for how UE nodes might develop. Feature requests that are too big to just be an issue. Related inspirations.

## Q&A

Ask a question! Answer a question! You know how this works. This is the place for "how do I do X?" or "why doesn't my workflow work?"

## How To

Solved a problem - maybe from the Q&A - and want to share how to do it? Put it here.

## General Discussion 

Anything else.

**Comment by imagin-eric** [comment]
This plug-in conflicts with efficiency-nodes-comfyui

**Reply by chrisgoringe** [reply]
More details would be helpful
